tests/integration/test_data.py

```diff
@@ -0,0 +1,34 @@
+import gzip
+import io
+import os
+import tarfile
+import tempfile
+import unittest
+
+import backend as F
+
+import dgl
+import dgl.data as data
+import numpy as np
+import pandas as pd
+import pytest
+import yaml
+from dgl import DGLError
+
+
+@unittest.skipIf(
+    F._default_context_str == "gpu",
+    reason="Datasets don't need to be tested on GPU.",
+)
+@unittest.skipIf(dgl.backend.backend_name == "mxnet", reason="Skip MXNet")
```

Comment:
Please double check whether pytest is a good option to run integration test like this.

Reply:
OK, I will later figure it out on DGL_scripts repo to see whether asv_run is a good choice.